### ISSUE TITLE
How to launch commands from terminal (Feature suggestion?)

### ISSUE THREAD

**Issue by Farravid** [glzr-io/glazewm#517]
Similar to i3-msg. Is there a way to tell glazewm to perform some commands.
For example reloading the configuration or moving some containers to workspaces.

That would be awesome in order to improve automatization of the setup and a lot of freedom for customization.

Thanks!!

**Comment by Lol-Lap** [comment]
> Similar to i3-msg. Is there a way to tell glazewm to perform some commands. For example reloading the configuration or moving some containers to workspaces.
> 
> That would be awesome in order to improve automatization of the setup and a lot of freedom for customization.
> 
> Thanks!!

Yeah, yO[window rule](https://github.com/glzr-io/glazewm#window-rulesl)

**Comment by Lol-Lap** [comment]
> Similar to i3-msg. Is there a way to tell glazewm to perform some commands. For example reloading the configuration or moving some containers to workspaces.
> 
> That would be awesome in order to improve automatization of the setup and a lot of freedom for customization.
> 
> Thanks!!

Yeah, you can customize that with the [window rule](https://github.com/glzr-io/glazewm#window-rulesl)

**Comment by Lol-Lap** [comment]
> Similar to i3-msg. Is there a way to tell glazewm to perform some commands. For example reloading the configuration or moving some containers to workspaces.
> 
> That would be awesome in order to improve automatization of the setup and a lot of freedom for customization.
> 
> Thanks!!

Yeah, you can customize that with the [window rule](https://github.com/glzr-io/glazewm#window-rules)

**Comment by txj-xyz** [comment]
You can use WSL2 to call EXE programs directly such as Edge

`/mnt/c/Program\ Files\ \(x86\)/Microsoft/Edge/Application/msedge.exe --profile-directory=Default`
